spec/lib/common/client/middleware/request/multipart_request.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
```

Comment:
Got it, my question is about why the code for the middleware is copied into the spec/lib directory.